astro.config.mjs

```diff
@@ -4,5 +4,6 @@ import image from "@astrojs/image";
 
 // https://astro.build/config
 export default defineConfig({
+  site: process.env.URL,
```

Comment:
Two suggestions:
- Astro/vite prefer `import.meta.env` over `process.env`
- If this is dynamically set by Netlify, then it might need a fallback when running standalone.

```suggestion
  site: import.meta.env.URL || 'http://localhost:3000',
```

Reply:
Based on playing with this, if site is not provided, Astro already falls back to localhost for you

Reply:
That said, happy to make the fallback more explicit!